react/components/ProductSpecifications/index.js

```diff
@@ -122,12 +122,26 @@ const ProductSpecifications = ({
         {specificationItems.map((specification, i) => (
           <tr key={i}>
             <td
-              className={`${handles.specificationItemProperty} w-50 b--muted-4 bb pa5`}
+              className={`${
+                handles.specificationItemProperty
+              } ${handles.specificationItemProperty +
+                '-' +
+                specification.property.replace(
+                  /\W+/g,
+                  '_'
+                )} w-50 b--muted-4 bb pa5`}
```

Comment:
Instead of creating new handles manually, we recommend you use the `applyModifiers` function from `vtex.css-handles`. Here is the documentation for it: https://github.com/vtex-apps/css-handles , there's an example in the README file.

Basically this function will take care of adding the `specification.property` value to the original CSS handle for you, much easier! 😄

Reply:
Yes much easier!!